doc/src/build/fullnode.md

```diff
@@ -94,9 +94,11 @@ Debian distributions and tests against
 That said, you are welcome to run a Sui fullnode on the operating system of your
 choosing and submit changes to accommodate that environment.
 
-Before building, ensure the required tools are installed in your environment as
-outlined in the [Prerequisites](../build/install.md#prerequisites) section. Note,
-you will fork the Sui repository here rather than clone it as described in
+Before building, ensure the required tools are installed and updated in your environment as
+outlined in the [Prerequisites](../build/install.md#prerequisites) section. In particular,
+ensure [Rust](../build/install.md#rust) is up-to-date.
+
+Note, you will fork the Sui repository here rather than clone it as described in
```

Comment:
A few lines below, we actually want this:

```shell
    $ apt-get update \
      && apt-get install --no-install-recommends \
      tzdata \
      git \
      ca-certificates \
      curl \
      build-essential \
      libssl-dev \
      pkg-config \
      libclang-dev \
      cmake
```

And we also want to mention "if you are using Windows (WSL), you will want to install a fresh copy of CLang/LLVM. The instructions may look very similar to the above if you are using Windows Subsystem for Linux (WSL), otherwise, please refer to the [CLang documentation](https://clang.llvm.org/)."

Reply:
> 

Excellent! Since we already link from:
https://docs.sui.io/build/install#microsoft-windows-specific
To:
https://releases.llvm.org/

What do you think of us instead moving this `apt-get` command to the central install docs and highlighting in the Fullnode doc that all major operating systems are now supported? I am trying to standardize between Install, Fullnode, and Contribute (where we fork):
https://docs.sui.io/contribute

It would be good to say the same thing everywhere and rely upon the central Install Sui page for the canonical base setup instructions.

Reply:
Makes sense to me. The edits I'm suggesting above are simply taking the command and removing the stuff that makes it suitable for use in a headless server (`NONINTERACTIVE`, `-y`): you typically want your instructions to assume that you have a human running the commands at the console, developers can figure out how to automate those instructions later.

The reason those small options are in there is they are copy-pasted from our Docker setup (where automated setup is the norm).

Reply:
> if you are using Windows (WSL), you will want to install a fresh copy of CLang/LLVM. The instructions may look very similar to the above if you are using Windows Subsystem for Linux (WSL), otherwise, please refer to the [CLang documentation](https://clang.llvm.org/).

OK, thanks. I've added the WSL references but am still wondering if they are needed given the example Linux command includes `libclang-dev`. Also, are we certain we don't need the `TZ=Etc/UTC` setting? Thanks for your help!

@randall-Mysten is to pick up the merge of install steps from fullnode.md to install.md in a separate change.

Reply:
Merging this in. Will let Randall continue in:
https://github.com/MystenLabs/sui/issues/3897